nodejs/eks/addon.ts

```diff
@@ -55,3 +55,25 @@ export class Addon extends pulumi.ComponentResource {
         this.addon = addon;
     }
 }
+
+// Sorts the keys of an object to ensure the output is deterministic.
+function stringifyReplacer(key: string, value: any) {
+    return value instanceof Object &&
+        !(value instanceof Array || value instanceof Date || value instanceof Function)
```

Comment:
nit: this ternary statement seems pretty complicated. Can be break it up to be more readable?